packages/tenant-process/src/services/tenantService.ts

```diff
@@ -261,6 +286,76 @@ export function tenantServiceBuilder(
       }
     },
 
+    async addCertifiedAttribute(
+      {
+        tenantId,
+        tenantAttributeSeed,
+        organizationId,
+        correlationId,
+      }: {
+        tenantId: TenantId;
+        tenantAttributeSeed: tenantApi.CertifiedTenantAttributeSeed;
+        organizationId: TenantId;
+        correlationId: string;
+      },
+      logger: Logger
+    ): Promise<Tenant> {
+      logger.info(
+        `Add certified attribute ${tenantAttributeSeed.id} to tenant ${tenantId}`
+      );
+
+      const requesterTenant = await retrieveTenant(
+        organizationId,
+        readModelService
+      );
+
+      const certifierId = getTenantCertifierId(requesterTenant.data);
+
+      const attribute = await retrieveAttribute(
+        unsafeBrandId(tenantAttributeSeed.id),
+        readModelService
+      );
+
+      if (attribute.kind !== attributeKind.certified) {
+        throw attributeNotFound(attribute.id);
+      }
+
+      if (!attribute.origin || attribute.origin !== certifierId) {
+        throw attributeDoesNotBelongToCertifier(
+          attribute.id,
+          organizationId,
+          tenantId
+        );
+      }
+
+      const targetTenant = await retrieveTenant(tenantId, readModelService);
+
+      // eslint-disable-next-line functional/no-let
+      let updatedTenant = await assignCertifiedAttribute({
+        targetTenant: targetTenant.data,
+        attribute,
+      });
+
+      updatedTenant = await revaluateTenantKind({
+        tenant: updatedTenant,
```

Comment:
let's avoid mutability when not strictly required
```suggestion
      const tenantWithNewAttribute = await assignCertifiedAttribute({
        targetTenant: targetTenant.data,
        attribute,
      });

      const updatedTenant = await revaluateTenantKind({
        tenant: tenantWithNewAttribute,
```